target/linux/ath79/image/generic.mk

```diff
@@ -860,6 +861,18 @@ define Device/engenius_ens202ext-v1
 endef
 TARGET_DEVICES += engenius_ens202ext-v1
 
+define Device/engenius_enstationac-v1
+  $(Device/engenius_loader_okli)
+  SOC := qca9557
+  DEVICE_MODEL := EnStationAC
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := ath10k-firmware-qca988x-ct-full-htt rssileds
```

Comment:
Please use ath10k-firmware-qca988x-ct like for the other devices. If you want to use a different package, you will have to pull it via opkg or build custom images.

Reply:
just wondering why one is the default over the other?

Reply:
Well, one has to be default. I assume the smallest package was taken ...